### ISSUE TITLE
telepresence: error: PANIC: assignment to entry in nil map

### ISSUE THREAD

**Issue by SunPj** [telepresenceio/telepresence#2753]
I got an error when I run following intercept command 
```
telepresence intercept my-service --port 9000:http --env-file /tmp/intercept.env
```

Output is 
```
Using Deployment namespace-service
telepresence: error: PANIC: assignment to entry in nil map

See logs for details (2 errors found): "/home/***/.cache/telepresence/logs/daemon.log"
If you think you have encountered a bug, please run `telepresence gather-logs` and attach the telepresence_logs.zip to your github issue or create a new one: https://github.com/telepresenceio/telepresence/issues/new?template=Bug_report.md .

```

**Versions (please complete the following information):**
 - Output of `telepresence version`

```
Client: v2.6.8 (api v3)
Root Daemon: v2.6.8 (api v3)
User Daemon: v2.6.8 (api v3)
```

 - Operating system of workstation running `telepresence` commands -> Archlinux
 - Kubernetes environment -> EKS (AWS)

Some logs
```
2022-08-24 17:23:26.5329 debug   connector/session/intercept-port-forward : Started forward for {Name:my-service PodIP:10.0.1.222}
2022-08-24 17:23:26.5329 debug   connector/session/intercept-port-forward : posting search paths [ambassador default kube-node-lease kube-public kube-system] and namespaces [default]
2022-08-24 17:23:26.5329 debug   connector/server-grpc/conn=6/CreateIntercept-13 : returned
2022-08-24 17:23:26.5330 info    connector/session/intercept-port-forward/10.0.1.222:40565 : Mounting file system for intercept "my-service" at "/tmp/telfs-1828395294"
2022-08-24 17:23:26.5335 debug   connector/session/intercept-port-forward : search paths posted successfully
2022-08-24 17:23:26.5337 debug   connector/server-grpc/conn=6/ResolveIngressInfo-14 : called
2022-08-24 17:23:26.5339 debug   connector/server-grpc/conn=2 : client Send loop started
2022-08-24 17:23:26.5340 debug   connector/server-grpc/conn=2 : manager Recv loop started
2022-08-24 17:23:26.5340 debug   connector/server-grpc/conn=2 : client Recv loop started
2022-08-24 17:23:26.5339 debug   connector/server-grpc/conn=2 : manager Send loop started
2022-08-24 17:23:26.5340 info    connector/session/intercept-port-forward/10.0.1.222:40565 : started command /usr/bin/sshfs sshfs -F none -f -C -oConnectTimeout=10 -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -o slave -o follow_symlinks -o allow_root localhost: /tmp/telfs-1828395294 : exec.pid="28079"
2022-08-24 17:23:26.5349 debug   connector/server-grpc/conn=7/UserNotifications-15 : called
2022-08-24 17:23:26.5349 debug   connector/server-grpc/conn=7/GetCloudAPIKey-16 : called
2022-08-24 17:23:26.5350 debug   connector/server-grpc/conn=7/GetCloudAPIKey-16 : returned
2022-08-24 17:23:26.5353 debug   connector/server-grpc/conn=7/UserNotifications-15 : returned
2022-08-24 17:23:26.7705 warning connector/session/ResolveIngressInfo-0 : Unexpected error tearing down systema connection: <nil>

```

**Comment by raphaelreyna** [comment]
You can try using v2.7.2; this could be related to one of the nil map panic bugs addressed in that release.

**Comment by thallgren** [comment]
Closing this. It's very likely fixed in 2.7.2 and no more info has been provided.
